src/services/sessionReports.ts

```diff
@@ -37,8 +37,10 @@ async function findSessionHelper(where: WhereOptions, plural = false): Promise<S
       'eventId',
       'data',
       'updatedAt',
+      [sequelize.literal('Date(CASE WHEN "SessionReportPilot".data->>\'startDate\' = \'\' THEN NULL ELSE "SessionReportPilot".data->>\'startDate\' END)'), 'startDate'],
```

Comment:
Also, don't use CASE for this, use NULLIF(<value>, ''), as this is far easier to read and is much faster